### PR TITLE
[changelog] Changelog for 0.15.17

### PR DESCRIPTION
**Pull request by arnikola**
Changelog for 0.15.17

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+# 0.15.17
+
+## Features 
+- **M3Query**: Add divideSeriesLists Graphite function ([#2584](https://github.com/m3db/m3/pull/2584))
+- **M3Query**: Add applyByNode Graphite function ([#2654](https://github.com/m3db/m3/pull/2654)) 
+- **M3Query**: Graphite ParseTime function support greatly expanded to be more in line with Graphite allowances ([#2621](https://github.com/m3db/m3/pull/2621)) 
+
+## Bug Fixes
+
+- **M3DB**: Fix a bug in bootstrap index cache that would cause long bootstrap tmu ([#2703](https://github.com/m3db/m3/pull/2703))
```
Comment (ryanhall07) [inline CHANGELOG.md:12]: tmu?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+# 0.15.17
+
+## Features 
+- **M3Query**: Add divideSeriesLists Graphite function ([#2584](https://github.com/m3db/m3/pull/2584))
```
Comment (robskillington) [inline CHANGELOG.md:6]: Should be aggregate function?

**Review by ryanhall07 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (arnikola) [reply CHANGELOG.md:12]: too much utime

Comment (notbdu) [inline CHANGELOG.md:12]: Is tmu supposed to be "times"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 ## Features 
-- **M3Query**: Add divideSeriesLists Graphite function ([#2584](https://github.com/m3db/m3/pull/2584))
+- **M3Query**: Add aggregate Graphite function ([#2584](https://github.com/m3db/m3/pull/2584))
 - **M3Query**: Add applyByNode Graphite function ([#2654](https://github.com/m3db/m3/pull/2654)) 
 - **M3Query**: Graphite ParseTime function support greatly expanded to be more in line with Graphite allowances ([#2621](https://github.com/m3db/m3/pull/2621)) 
 
 ## Bug Fixes
 
-- **M3DB**: Fix a bug in bootstrap index cache that would cause long bootstrap tmu ([#2703](https://github.com/m3db/m3/pull/2703))
+- **M3DB**: Fix a bug in bootstrap index cache that wougitld cause long bootstrap times ([#2703](https://github.com/m3db/m3/pull/2703))
```
Comment (ryanhall07) [inline CHANGELOG.md:12]: typo

**Pull request merged**